client/src/main/java/com/vesoft/nebula/AbstractClient.java

```diff
@@ -0,0 +1,102 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula;
+
+import com.facebook.thrift.TException;
+import com.facebook.thrift.protocol.TProtocol;
+import com.facebook.thrift.transport.TTransport;
+import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
CheckStyle would probably remind me to move the static import to the top, just below package. not pretty sure though :(

Reply:
yep it's mistake